config/common.ts

```diff
@@ -0,0 +1,25 @@
+import { SidebarsConfig } from "@docusaurus/plugin-content-docs";
+
+const sidebarType: SidebarsConfig = {
+  sidebar: [],
+};
+
+export const sidebar: typeof sidebarType.sidebar = [
+  {
```

Comment:
This is completely awful. I don't know how to make it better though (and am pretty new to typescript). The type needed isn't exported, so this hack is used instead.